_docs/style/typography.md

```diff
@@ -29,61 +23,167 @@ body text. Hierarchy can be established using size and
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Container Title
+{: .hxContainerTitle }
+Container Titles are good for titling within elements like cards, modals, etc.
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% include type-styles.html htmlexample='<h4 class="hxContainerTitle">Container Title</h4>' typeface="Roboto" weight="Medium (500)" fontsize="16px / 1rem" csscode="Gray 900" hex="#424242" linespace="1" bottomspace="24px / 1.5rem" %}
+{% endcolumn %}
+</div>
 
-## Styles
 <div class="hxRow">
-{% include type-styles.html htmlexample='<span class="display-font" style="color: #424242">Display</span>' name="Display" typeface="Roboto" weight="Thin (100)" size="48px / 3.0em" csscode="Grey 900" hex="#424242" bottomspace="32px / 2em" %}
-{% include type-styles.html htmlexample='<h1 class="headerDisplayList">Main Heading</h1>' name="H1" typeface="Roboto" weight="Regular (400)" size="24px / 1.5em" csscode="Grey 900" hex="#424242" bottomspace="24px / 1.5em" %}
+{% column left:"hxCol-8" %}
+### Body Styles
+{: .hxSubSectionTitle }
+{% endcolumn %}
 </div>
 
 <div class="hxRow">
-{% include type-styles.html htmlexample='<h1 class="headerDisplayList header-alt">Light Heading</h1>' name="H1 Light" typeface="Roboto" weight="Thin (100)" size="24px / 1.5em" csscode="Grey 900" hex="#424242" bottomspace="24px / 1.5em" %}
-{% include type-styles.html htmlexample='<h2 class="headerDisplayList">Subheading</h2>' name="H2" typeface="Roboto" weight="Regular (400)" size="20px / 1.25em" csscode="Grey 900" hex="#424242" bottomspace="12px / 0.75em" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Body Text
+{: .hxContainerTitle }
+Body text is the default style and should be used across the interface for most information.
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% include type-styles.html htmlexample='<p>Body Text</p>' typeface="Roboto" weight="Regular (400)" fontsize="16px / 1rem" csscode="Gray 900" hex="#424242" bottomspace="24px / 1rem" linespace="1.5" %}
+{% endcolumn %}
 </div>
 
 <div class="hxRow">
-{% include type-styles.html htmlexample='<h3 class="headerDisplayList">Subheading</h3>' name="H3" typeface="Roboto" weight="Medium (500)" size="16px / 1.25em" csscode="Grey 900" hex="#424242" bottomspace="12px / 0.75em" %}
-{% include type-styles.html htmlexample='<span class="body1 headerDisplayList">Body 1</span>' name="Body" typeface="Roboto" weight="Regular (400)" size="16px / 1em" csscode="Grey 900" hex="#424242" linespace="24px / 1.5em" bottomspace="24px/1.5em" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Subdued Style
+{: .hxContainerTitle }
+Subdued style can be applied to any text 14px and taller, and is used to let more important text stand out. It is not the same as an inactive link or button.
```

Comment:
You can use the subdued style with any text 14px and larger to help important text stand out. The subdued style is not the same as an inactive link or button.